packages/notification/docs/InlineNotification.stories.tsx

```diff
@@ -49,245 +20,134 @@ export default {
+            open until complete. <a href="/">View all</a>
+          </InlineNotification>
+        </StoryWrapper.Row>
+        <StoryWrapper.Row rowTitle="Persistent">
+          <InlineNotification
+            type="cautionary"
+            title="Cautionary title"
+            persistent
+          >
+            The syncing process can take some time to complete. Keep this window
+            open until complete. <a href="/">View all</a>
+          </InlineNotification>
+        </StoryWrapper.Row>
+        <StoryWrapper.Row rowTitle="Multiline">
+          <InlineNotification
+            type="positive"
+            title="Positive title"
+            forceMultiline
+          >
+            {multilineText}
+          </InlineNotification>
+        </StoryWrapper.Row>
+        <StoryWrapper.Row rowTitle="Forced Multiline">
+          <InlineNotification
+            type="negative"
+            title="Negative title"
+            forceMultiline
+          >
+            {forceMultilineText}
+          </InlineNotification>
+        </StoryWrapper.Row>
+      </StoryWrapper>
+    </>
+  )
+}
 
-export const DismissibleMultiline = () => (
-  <InlineNotification
-    type="negative"
-    title="Negative"
-    automationId="notification1"
-  >
-    {multilineText}
-  </InlineNotification>
-)
-DismissibleMultiline.storyName = "Dismissible, Multiline"
-DismissibleMultiline.parameters = { chromatic: { disable: false } }
+export const StickerSheetDefault = StickerSheetTemplate.bind({})
+StickerSheetDefault.storyName = "Sticker Sheet (Default)"
+StickerSheetDefault.parameters = { chromatic: { disable: false } }
 
-export const DismissibleForcedMultiline = () => (
-  <InlineNotification
-    type="negative"
-    title="Negative"
-    automationId="notification1"
-    forceMultiline
-  >
-    {forceMultilineText}
-  </InlineNotification>
-)
-DismissibleForcedMultiline.storyName = "Dismissible, Forced Multiline"
-
-export const DismissibleSlim = () => (
-  <InlineNotification
-    type="positive"
-    title="Success"
-    automationId="notification1"
-  >
-    <a href="/">Manage users is now available</a>
-  </InlineNotification>
-)
-DismissibleSlim.storyName = "Dismissible, Slim"
+export const StickerSheetReversed = StickerSheetTemplate.bind({})
+StickerSheetReversed.storyName = "Sticker Sheet (Reversed)"
+StickerSheetReversed.args = { isReversed: true }
+StickerSheetReversed.parameters = {
+  backgrounds: { default: "Purple 700" },
+  chromatic: { disable: false },
+}
 
-export const PersistentPositive = () => (
+export const AutohideDemo = props => (
   <InlineNotification
     type="positive"
     title="Success"
-    persistent
-    automationId="notification1"
+    {...props}
+    autohide
+    autohideDelay="short"
   >
     New user data, imported by mackenzie@hooli.com has successfully uploaded.{" "}
     <a href="/">Manage users is now available</a>
   </InlineNotification>
```

Comment:
```suggestion
  </InlineNotification>
  <p>Content below the notification</p>
```

So that it doesn't look like it goes completely blank :)